napari/layers/image/image.py

```diff
@@ -517,12 +524,64 @@ def interpolation(self, interpolation):
         self.events.interpolation(value=self._interpolation[self._ndisplay])
 
     @property
-    def experimental_slicing_plane(self):
-        return self._experimental_slicing_plane
+    def depiction(self):
+        """The current 3D depiction mode.
+
+        Selects a preset depiction mode in vispy
+            * volume: images are rendered as 3D volumes.
+            * plane: images are rendered as 2D planes embedded in 3D.
+        """
+        return str(self._depiction)
+
+    @depiction.setter
+    def depiction(self, depiction: Union[str, VolumeDepiction]):
+        """Set the current 3D depiction mode."""
+        self._depiction = VolumeDepiction(depiction)
+        self._update_plane_callbacks()
+        self.events.depiction()
+
+    def _reset_plane_parameters(self):
+        """Set plane attributes to something valid."""
+        self.plane.position = np.array(self.data.shape) / 2
+        self.plane.normal = (1, 0, 0)
+
+    # The following callbacks are added as partials otherwise they are
+    # considered as methods on the class, this is similar to this problem:
+    # https://stackoverflow.com/questions/40338652/how-to-define-enum-values-that-are-functions
+    _plane_drag_callback = partial(move_plane_along_normal)
+    _plane_double_click_callback = partial(set_plane_position)
```

Comment:
Why exactly is this a problem?

Reply:
if the functions are put on the class as `Layer._plane_drag_callback = move_plane_along_normal` then `self` is passed to the 'method' along with the specified arguments when it is called -> this way you can call the function at `Layer._plane_drag_callback()` without modifying its signature

Reply:
Sorry, I don't understand this part of the code. What is the behavior you aiming for? From the comments, it sounds like you might want a class method, but I'm a little confused about what the goal is.

Reply:
If the function doesn't need the layer instance at self, I'm confused about why this isn't just a function? `if move_plane_along_normal in self.mouse_drag_callbacks`?

Reply:
@alisterburt I don't think this is blocking, but I do hate the massive number of private attributes we've got on our classes, so if we merge without a more elegant solution, could you please create an issue summarising this discussion and reminding us to investigate?